com.unity.shadergraph/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - Support for the XboxSeries platform has been added.
   - Stereo Eye Index, Instance ID, and Vertex ID nodes added to the shadergraph library.
   - Added View Vector Node doc
+  - Added `Calculate Level Of Detail Texture 2D` node, for calculating a Texture2D LOD level.
+  - Added `Gather Texture 2D` node, for retrieving the four samples (red component only) that would be used for bilinear interpolation when sampling a Texture2D.
+  - Added an Integer slot control view.
```

Comment:
Might be better to keep this out of the changelog if it's not a feature the users can access.

Reply:
I removed this.